content/en/docs/contribute/new-content/new-features.md

```diff
@@ -95,14 +95,16 @@ deadlines.
 
 ### Open a placeholder PR
 
-1. Open a pull request against the
+1. Open a **draft** pull request against the
 `dev-{{< skew nextMinorVersion >}}` branch in the `kubernetes/website` repository, with a small
-commit that you will amend later.
+commit that you will amend later. To create a draft pull request, use the
+Create Pull Request drop-down and select **Create Draft Pull Request**,
+then click **Draft Pull Request**.
```

Comment:
Okay, now I see. Draft Pull Request is something one can raise from Web UI. If I'm opening a PR from git command line, there is actually no difference between a "draft" and a "formal" one, right?

Reply:
Yes, that's correct. This is specific to Github not git.
It should not impact the people using the git command line to make a pull request, they'll have to use the UI to set it to draft if needed.

Reply:
Thanks for the clarification. @annajung

Reply:
I think the GitHub command line tools do let you mark a PR as draft, and also to mark it ready for review: [`gh pr create --draft`](https://cli.github.com/manual/gh_pr_create).